### PR TITLE
Feature/boiler platify finder

### PR DESCRIPTION
**Pull request by JoeKer1**
This sets up finder for its new angular boilerplate foundation
[Must be merged with this pr](https://github.com/crdschurch/crds-connect/pull/707)

### CODE REVIEW

File: apps/crossroads_interface/web/templates/legacy/body.html.eex [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <script type="text/javascript" src="https://s7.addthis.com/js/300/addthis_widget.js#defer=1" defer></script>
 <script type="text/javascript" src="https://www.gstatic.com/cv/js/sender/v1/cast_sender.js"></script>
 <script src="https://use.fontawesome.com/ccd1aba2f7.js"></script>
-
+<script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```
Comment (danrye) [inline apps/crossroads_interface/web/templates/legacy/body.html.eex]: how come these changes also snuck into this PR?

File: apps/crossroads_interface/web/templates/shared/common_body.html.eex [hunk 2]
```diff
@@ -7,7 +7,6 @@
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.3/jquery.min.js"></script>
     <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
-    <script type="text/javascript" src="https://js.stripe.com/v2/"></script>
```
Comment (danrye) [inline apps/crossroads_interface/web/templates/shared/common_body.html.eex]: how come these changes also snuck into this PR?

File: apps/crossroads_interface/web/templates/legacy/body.html.eex (hunk 1 above)
Comment (JoeKer1) [reply apps/crossroads_interface/web/templates/legacy/body.html.eex]: I merged the branch to test it in runway

File: apps/crossroads_interface/web/templates/shared/common_body.html.eex (hunk 2 above)
Comment (JoeKer1) [reply apps/crossroads_interface/web/templates/shared/common_body.html.eex]: I merged the branch to test it in runway

**Pull request merged**